app/graphql/resolvers/freak_create_resolver.rb

```diff
@@ -6,12 +6,15 @@ class FreakCreateResolver < Resolvers::Base
     argument :last_name, String, required: true
     argument :description, String, required: true
     argument :email, String, required: true
+    argument :norm_id, GraphQL::Types::ID, required: true
 
     type [Types::FreakType], null: false
 
     # :reek:UtilityFunction
     def resolve(params)
-      Freak.create(params)
+      norm = Norm.find(params[:norm_id])
```

Comment:
What would happen if the `norm_id` does not exist?

Reply:
I think app will break:((

Reply:
Indeed. Normally we'd have to validate that the `norm_id` first exists before trying to assign it to a freak.

At this point I don't think it's necessary, but please make a ticket to track this & add a `# TODO:` note comment here stating that we'll have to verify if the `norm` exists.

Reply:
I don't know